packages/devtools_app/lib/src/tree.dart

```diff
@@ -56,21 +57,30 @@ class _TreeViewState<T extends TreeNode<T>> extends State<TreeView<T>>
   @override
   void initState() {
     super.initState();
-    _initData();
-    _updateItems();
+    widget.dataRootsListenable.addListener(_updateTreeView);
```

Comment:
can you use addAutoDisposeListener instead?

Reply:
Done